include/vctr/Expressions/ExpressionChainBuilder.h

```diff
@@ -46,10 +46,16 @@ class RuntimeArgChain
 
     constexpr RuntimeArgChain()
     requires (sizeof... (ArgTuples) == 1)
-    : chain {{ std::tuple<>() }}
+        : chain {{ std::tuple<>() }}
     {}
 
-    static consteval size_t size() { return sizeof... (ArgTuples); }
+    // TODO: This is a workaround for https://github.com/sonible/VCTR/issues/111 and should be replaced by a generic solution
+    constexpr RuntimeArgChain()
+    requires (sizeof... (ArgTuples) == 3)
+        : chain {{ std::tuple<>() }, { std::tuple<>() }, { std::tuple<>() }}
+    {}
```

Comment:
This is a quick hardcoded solution for the case above where we need to support 3 empty tuples and a constructor without arguments. Should be replaced by a generic solution in the long run, which could be a bit tricky to get right, so I decided to go for the non-generic solution for now